### ISSUE TITLE
Dialyzer warnings introduced in *gettext_noop macros

### ISSUE THREAD

**Issue by jclem** [elixir-lang/gettext#146]
Since 1ae3b0c75beda486ec7ba6dced31350c1b6006ca, Dialyzer emits the following warning when using Gettext:

```
The inferred return type of 'MACRO-ngettext_noop'/3 ({{'.',[],['Elixir.Dollar.Gettext' | 'dngettext_noop',...]},[],[any(),...]}) has nothing in common with binary(), which is the expected return type for the callback of 'Elixir.Gettext.Backend' behaviour
```

This happens in both Elixir 1.3.4 and 1.4 on OTP 19.2 (have not tested other OTPs). A test repo (generated w/Elixir 1.4) can be used [here](https://github.com/jclem/gettext_test) to reproduce this.

To reproduce, compile the project and run dialyzer.

Currently I have `@dialyzer [{:nowarn_function, "MACRO-ngettext_noop": 3}]` in my module using Gettext to get around this.

I am, unfortunately, not dialyzer-savvy enough to have any idea what this warning means, so I'm hoping someone else recognizes it :)

**Comment by xtian** [comment]
I believe this is fixed in e8af08d8f669b8630d16ffb924c487f973c1ceee. Would it be possible to release 0.13.1 with this fix?

**Comment by whatyouhide** [comment]
I believe it was indeed fixed in that commit. @jclem could you give master a try so that if everything works well I can release 0.13.1? :) 💟

**Comment by jclem** [comment]
@whatyouhide Yup! Looks fixed. See https://github.com/jclem/gettext_test/tree/gettext-39b0f22a.

**Comment by whatyouhide** [comment]
I just released v0.13.1. Thanks for the help!
